MatrixSDK/Contrib/Swift/MXRestClient.swift

```diff
@@ -37,6 +37,18 @@ public enum MXAccountDataType: Equatable, Hashable {
     }
 }
 
+/// Represents the threads request include parameter
+public enum MXThreadsIncludeParameter: Equatable, Hashable {
+    case all
+    case participated
+    
+    var rawValue: String {
+        switch self {
+        case .all:               return kMXThreadsListIncludeAllParameter
+        case .participated:      return kMXThreadsListIncludeParticipatedParameter
```

Comment:
Wouldn't call this variable rawValue because this enum is not really conforming to the RawRepresentable protocol.
You could either have it conform to the protocol:
`public enum MXThreadsIncludeParameter: String`
or in case is not possible (since you are using cases that are non string literals) you can create the RawRepresentable on your own
public enum MXThreadsIncludeParameter: RawRepresentable, Equatable, Hashable {
  associatedType RawValue = String

  // declare the cases as you already did
  
  // implement rawValue as you already did
  // implement the init(rawValue: )
}

Otherwise just change to stringIdentifier.